dff/context_storages/json.py

```diff
@@ -6,79 +6,173 @@
 store and retrieve context data.
 """
 import asyncio
-from typing import Hashable
+from pathlib import Path
+from base64 import encodebytes, decodebytes
+from typing import Any, List, Set, Tuple, Dict, Optional
+
+from pydantic import BaseModel
+
+from .serializer import DefaultSerializer
+from .context_schema import ContextSchema, ExtraFields
+from .database import DBContextStorage, threadsafe_method, cast_key_to_string
 
 try:
-    import aiofiles
-    import aiofiles.os
+    from aiofiles import open
+    from aiofiles.os import stat, makedirs
+    from aiofiles.ospath import isfile
 
     json_available = True
 except ImportError:
     json_available = False
 
-from pydantic import BaseModel, model_validator
 
-from .database import DBContextStorage, threadsafe_method
-from dff.script import Context
+class SerializableStorage(BaseModel, extra="allow"):
+    pass
```

Comment:
This class is only used for context serializing using pydantic, since `Context` class doesn't inherit `BaseModel` itself and being able to serialize types like timestamp is very useful.